frontend/src/modules/modals/MultiStepWizard.tsx

```diff
@@ -827,20 +768,19 @@ const MultiStepWizard = (): JSX.Element => {
               </Grid>
             </Grid>
             <Grid item className={classes.marginContent}>
-              {multiParams.map((param: Data) => {
+              {multiParams.map((param: InternalState) => {
                 if (param.isSetvalEnabled) {
-                  if (param.setValue !== param.setValueActual) {
+                  if (param.ValueDraft !== param.ValueCurrent) {
                     return (
-                      <Typography variant="subtitle1">{`Change ${param.label} to ${param.setValue} ${param.units}?`}</Typography>
+                      <Typography variant="subtitle1">{`Change ${param.label} to ${param.ValueDraft} ${param.units}?`}</Typography>
                     );
                   }
                 } else if (
-                  param.alarmValues.length &&
-                  (param.alarmValues[0] !== param.alarmValuesActual[0] ||
-                    param.alarmValues[1] !== param.alarmValuesActual[1])
+                  param.alarmValues.lower !== param.alarmValuesActual.lower ||
+                  param.alarmValues.upper !== param.alarmValuesActual.upper
                 ) {
                   return (
-                    <Typography variant="subtitle1">{`Change ${param.label} alarm range to ${param.alarmValues[0]} - ${param.alarmValues[1]}?`}</Typography>
+                    <Typography variant="subtitle1">{`Change ${param.label} alarm range to ${param.alarmValues.lower} - ${param.alarmValues.upper}?`}</Typography>
```

Comment:
While we're at it, let's change `alarm range` to `alarm limits` in this text, and let's use the same format for displaying the range here as what we use in the event log (it involves square brackets, and it should include units). It would be great if we could display the old range, too: e.g. Change HR alarm limits from [0 BPM - 200 BPM] to [50 BPM - 100 BPM]?